src/components/Page.js

```diff
@@ -1,22 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import Sidebar from './Sidebar';
+import MobileMenuBar from './MobileMenuBar';
+import { mediaQueries } from './Common';
 
 const Container = styled.div`
   display: flex;
   align-items: stretch;
   min-height: 100vh;
 `;
 
+const LeftColumn = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+`;
+
 const Content = styled.div`
   padding: 24px 60px;
   box-sizing: border-box;
   width: 100%;
 `;
 
-export default ({ children }) => (
-  <Container>
-    <Sidebar />
-    <Content>{children}</Content>
-  </Container>
-);
\ No newline at end of file
+
+
+const Page = ({ children }) => {
+  const SidebarContainer = styled.div`
+    min-height: 100%;
+    border-right: 1px solid rgba(255, 255, 255, 0.3);
```

Comment:
why did this get moved from `src/components/Sidebar`?

Reply:
oop i moved it up before i realized i could just pass in the prop, moving it back down now